packages/fast-usdc-contract/src/exos/settler.ts

```diff
@@ -359,7 +360,7 @@ export const prepareSettler = (
           log('forwarding', fullValue, 'to', EUD, 'for', txHash);
           const dest: AccountId | null = (() => {
             try {
-              return chainHub.resolveAccountId(EUD);
+              return chainHub.resolveAccountId(EUD as Bech32Address);
```

Comment:
what justifies this cast? Why not use `Bech32Address` as the type of the `EUD` arg?